src/lib/components/workflow-actions.svelte

```diff
@@ -200,14 +187,23 @@
       return 'Resetting workflows is not enabled, please contact your administrator for assistance.';
     if (resetEnabled && workflow?.pendingChildren?.length > 0)
       return 'Cannot reset workflows with pending children.';
+    if (
```

Comment:
I would say that this entire set of conditions should be broken out into a utility function and unit tested. We can (and should) make that a task.

Reply:
Made a task, DT-969